mass_mailing_custom_unsubscribe/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+====================================================
+Customizable unsubscribe link on mass mailing emails
+====================================================
+
+With this module you can set a custom unsubscribe link append at bottom of mass
+mailing emails.
+
+
+Configuration
+=============
+
+To configure unsubscribe label go to Setting > Technical > Parameters > System parameters
+and add a 'mass_mailing.unsubscribe.label' parameter with html to set at bottom
+of mass emailing emails. Including '%(url)s' variable where unsubscribe link
+
+For example:
+```
+<small>You can unsubscribe <a href="%(url)s">here</a></small>
+
+```
+
+Additionally, you can disable this link if you set this parameter to 'False'
+
+If this parameter (mass_mailing.unsubscribe.label) is not set (or set to '')
+default 'Click to unsubscribe' link will appear. This default text is
+translatable via Settings > Translations > Application Terms > Translated terms
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/205/8.0
+
+
+Known issues / Roadmap
+======================
+
+* This custom html is not translatable, so as a suggestion, you can define
+the same text in several languages in several lines. For example:
+
+For example:
+```
+<small>[EN] You can unsubscribe <a href="%(url)s">here</a></small><br/>
+<small>[ES] Puedes darte de baja <a href="%(url)s">aquí</a></small>
```

Comment:
Yes, it's like a product in a supermarket, when manufacter indicates ingredients in several languages